common/src/sql/dbinit.sql

```diff
@@ -2309,6 +2309,8 @@ CREATE TABLE omicron.public.switch_port (
     )
 );
 
+CREATE INDEX ON switch_port (port_settings_id);
```

Comment:
After revisting https://rfd.shared.oxide.computer/rfd/0267, it doesn't seem wrong to let `switch_port`s share `port_settings`, especially since we will allow the sharing of `port_settings` via `groups`.

Reply:
Port settings are designed to be shared, so common settings can be applied to multiple ports. Because port settings can themselves include other port settings, this makes it possible to have baseline settings that apply to a large group of ports in combination with more specific settings that apply to only a small number of ports or a single port.